### PR TITLE
Adds BANNED message to antag panel

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
The Enabled in Prefs/Disabled in Prefs message will now show a different message if the person in question is banned from the antagonist in question.

oops.jpg

## Changelog
:cl:
admin: Traitor panel will now show if the person in question is banned from a specific antagonist.
/:cl:

### CODE REVIEW

**Comment by ItsTisme** [comment]
While I like the change, admins can just "create antag" so for that purpose its little moot

**Comment by francinum** [comment]
> 
> 
> While I like the change, admins can just "create antag" so for that purpose its little moot

Except not really, you can't control the amount of antags you create via CA, and overall this is a good idea.

File: code/modules/admin/antag_panel.dm [hunk 1]
```diff
@@ -166,7 +166,10 @@ GLOBAL_VAR(antag_prototypes)
 				pref_source = prototype
 				break
 		if(pref_source.job_rank)
-			antag_header_parts += pref_source.enabled_in_preferences(src) ? "Enabled in Prefs" : "Disabled in Prefs"
+			if(!is_banned_from(src.key, pref_source.job_rank))
+				antag_header_parts += pref_source.enabled_in_preferences(src) ? "Enabled in Prefs" : "Disabled in Prefs"
+			else
+				antag_header_parts += "BANNED from Role"
```
Comment (francinum) [inline code/modules/admin/antag_panel.dm]:
```suggestion
				antag_header_parts += "<span class='bad'><b>[BANNED]</b></span>"
```

**Review by francinum — DISMISSED** [review]

File: code/modules/admin/antag_panel.dm [hunk 2]
```diff
@@ -166,7 +166,10 @@ GLOBAL_VAR(antag_prototypes)
 				pref_source = prototype
 				break
 		if(pref_source.job_rank)
-			antag_header_parts += pref_source.enabled_in_preferences(src) ? "Enabled in Prefs" : "Disabled in Prefs"
+			if(!is_banned_from(src.key, pref_source.job_rank))
+				antag_header_parts += pref_source.enabled_in_preferences(src) ? "Enabled in Prefs" : "Disabled in Prefs"
+			else
+				antag_header_parts += "<span class='bad'><b>[BANNED]</b></span>"
```
Comment (francinum) [inline code/modules/admin/antag_panel.dm]:
```suggestion
				antag_header_parts += "<span class='bad'><b>\[BANNED\]</b></span>"
```

**Review by francinum — APPROVED** [review]

**Comment by ike709** [comment]
amazing

**Pull request merged**